addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -366,10 +366,18 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
      */
     start: function () {
         const proms = [this._super(...arguments)];
+        const isColumn = el => el.tagName === "DIV" && el.parentElement.classList.contains("row");
+        const isEditableFormColumn = el => isColumn(el) && !el.closest(".s_website_form_field, .s_website_form_submit");
         // Disable text edition
         this.$target.attr('contentEditable', false);
-        // Make button, description, and recaptcha editable
-        this.$target.find('.s_website_form_send, .s_website_form_field_description, .s_website_form_recaptcha').attr('contentEditable', true);
+        // Identifiy editable elements of the form: buttons, description,
+        // recaptcha and columns which are not fields.
+        const formEditableSelector = ".s_website_form_send, .s_website_form_field_description, .s_website_form_recaptcha";
+        for (const formEditableEl of this.$target[0].querySelectorAll(`:scope ${formEditableSelector}, .row > div`)) {
+            if (!isColumn(formEditableEl) || isEditableFormColumn(formEditableEl)) {
+                formEditableEl.setAttribute("contentEditable", true);
```

Comment:
Nitpicking: the attribute expects a string, put "true" instead of letting it be cast implicitly. You can then actually use the property using `.contentEditable = "true"`